features/seeds/seed_data.rb

```diff
@@ -38,6 +39,44 @@ def uri
   def data
     path_to_data_file = File.join(__dir__, "fixtures", file_name)
 
-    File.read(path_to_data_file)
+    data_content = File.read(path_to_data_file)
+
+    inflate_content(data_content)
+  end
+
+  def inflate_content(data)
+    data = JSON.parse(data)
```

Comment:
Add comment: `# Transform a string containing JSON format to an hash object (ex: { a: 1 })`